debian/swss.install

```diff
@@ -1,8 +1,7 @@
 swssconfig/sample/netbouncer.json etc/swss/config.d
 swssconfig/sample/00-copp.config.json etc/swss/config.d
-swssconfig/sample/msn2700.32ports.buffers.json etc/swss/config.d
+swssconfig/sample/msn27xx.32ports.buffers.json.j2 usr/share/sonic/templates
```

Comment:
i'm also wondering if we could still leave the file inside the `/etc/swss/config.d` folder. i think to have all swss related configurations (including templates) inside this folder is better than putting the files somewhere around.

Reply:
probably you are right...
also the source directory looks not very nice
"production" configs should be under some `sonic-buildimage/platform/<vendor>/config` and `sonic-buildimage/platform/generic/config`
I think this should be done globally for all configuration files

Reply:
i agree with your idea here. i created the `sample` folder but haven't imagined that it is no longer 'sample' folder. we could fix it in the long run. but right now, i'd suggest it is okay to put it into the 'sample' folder.